ODIN_II/SRC/subtractions.cpp

```diff
@@ -396,10 +396,12 @@ void split_adder_for_sub(nnode_t *nodeo, int a, int b, int sizea, int sizeb, int
 	for(i = 0; i < b; i++)
 	{
 		not_node[i] = allocate_nnode();
```

Comment:
just skip the allocation entirely and set not_node[i] to NULL rather since it will be lost in every condition